src/status_im2/contexts/syncing/how_to_pair/view.cljs

```diff
@@ -0,0 +1,131 @@
+(ns status-im2.contexts.syncing.how-to-pair.view
+  (:require
+    [status-im2.contexts.syncing.how-to-pair.style :as style]
+    [status-im2.common.resources :as resources]
+    [utils.i18n :as i18n]
+    [quo2.core :as quo]
+    [quo2.foundations.colors :as colors]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]))
+
+(defn list-icon
+  [s]
+  [rn/view style/list-icon
+   [quo/text style/list-icon-text s]])
+
+(defn mobile
+  []
+  [rn/scroll-view style/scroll-view
+   [quo/text style/paragraph (i18n/label :t/signing-in-from-another-device)]
+   [rn/image
+    {:source (resources/get-image :mobile-how-to-pair-sign-in)
+     :style  style/image}]
+   [rn/view style/numbered-list
+    [rn/view style/numbered-list-item
+     [list-icon 1]
```

Comment:
Minot, but for consistency, I'd suggest to pass a string value or change `list-icon` param name

Reply:
done